fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -7257,4 +7259,46 @@ public static boolean isStoredTableNamesLowerCase() {
     public static boolean isTableNamesCaseInsensitive() {
         return GlobalVariable.lowerCaseTableNames == 2;
     }
+
+    public void compactTable(AdminCompactTableStmt stmt) throws DdlException {
+        String dbName = stmt.getDbName();
+        String tableName = stmt.getTblName();
+
+        String type = stmt.getCompactionType();
+        if (type == null || (!type.equals("base") && !type.equals("cumulative"))) {
```

Comment:
This check should be done in analysis phase

Reply:
> This check should be done in analysis phase

OK, thank you.